tools/ui-components/src/form-control/form-control-feedback.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { FormControlVariationProps } from './types';
+
+export const FormControlFeedback = ({
+  children,
+  className,
+  testId
+}: FormControlVariationProps) => {
+  const defaultClasses =
+    'absolute top-0 right-0 z-2 block w-8 h-8 leading-8 ' +
```

Comment:
most of the spacing is done with rm base on our html font size (18), I tried to find the closest pre-defined styles.